enterprise/backend/src/metabase_enterprise/task/cache.clj

```diff
@@ -257,9 +266,10 @@
     (-> (.getFireTimeAfter cron (t/java-date now))
         (t/offset-date-time (t/zone-offset)))))
 
-(defn- refresh-cache-configs!
+(defenterprise refresh-cache-configs!
```

Comment:
Do you think it would be helpful to have more info about "what happened" from this function? Since we either `refresh-schedule-cache!` each cache-config that's ready to run or not, perhaps having something like `{:refreshed <int> :total <int>}` as a return would be helpful, especially to test for example this comment: `;; Manually expire the existing cache entry. Now the cache should be refreshable`